model/src/gateway/payload/member_chunk.rs

```diff
@@ -117,55 +118,27 @@ impl<'de> Visitor<'de> for MemberChunkVisitor {
                         return Err(DeError::duplicate_field("presences"));
                     }
 
-                    presences = Some(map.next_value()?);
+                    let deserializer = PresenceMapDeserializer::new(GuildId(0));
```

Comment:
is this supposed to always use id 0?

Reply:
I'll make a note about it but yes, since the guild ID may not be deserialised yet it'll use a placeholder value, and it updates it a little below the loop

Reply:
alright just wanted to double check on this, a note's a good idea.

Reply:
Added the note